### ISSUE TITLE
nginx pod CrashLoopBackOff

### ISSUE THREAD

**Issue by wrkode** [grafana/mimir#1669]
**Don't know if this is a bug** _using blank template_

**Condition**
Nginx pod errors out and goes in CrashLoopBackOff

**Pod logs**
```
/docker-entrypoint.sh: No files found in /docker-entrypoint.d/, skipping configuration
2022/04/10 07:38:01 [emerg] 1#1: host not found in resolver "kube-dns.kube-system.svc.cluster.local" in /etc/nginx/nginx.conf:25
nginx: [emerg] host not found in resolver "kube-dns.kube-system.svc.cluster.local" in /etc/nginx/nginx.conf:25
```

**kube-dns resolution check**
```
kubectl exec -i -t dnsutils -- nslookup mimir-distributed-0-1649575929-distributor
Server:		10.43.0.10
Address:	10.43.0.10#53

Name:	mimir-distributed-0-1649575929-distributor.default.svc.cluster.local
Address: 10.43.99.10
```

**Environment details**
Kubernetes engine: RKE2 version v1.21.7+rke2r2

**Comment by wrkode** [comment]
Thanks @pracucci, I don't see a link/way to join the Grafana Slack.
I'll try the override option and inform on this issue.
I'm running on RKE2, not EKS, but I'll check the troubleshooting link as well

**Comment by wrkode** [comment]
@pracucci changing the resolver name solved the issue. And yes, just joined the Slack channel. thanks a million.
I do believe that there's room to expand/improve the mimir-distributed documentation. I'd be happy to help.

**Comment by pracucci** [comment]
> I do believe that there's room to expand/improve the mimir-distributed documentation especially around Ingress. I'd be happy to help.

Yes, please! Could you coordinate with @krajorama, please? He's `@György Krajcsovits` on Slack.

**Comment by pracucci** [comment]
_Going to close this issue since has been solved._
